### PR TITLE
feat: add logic to migration to count errors by model

### PR DESCRIPTION
**Pull request by nathanielc**

### CODE REVIEW

File: event-svc/src/event/migration.rs [hunk 1]
```diff
@@ -379,13 +432,54 @@ pub enum Error {
...
         Self::from(err.into())
     }
 }
 
+trait WithModelContext {
+    type Target;
+    fn with_model_context(self, model: &ModelContext) -> Self::Target;
+}
+
+impl<T, E> WithModelContext for std::result::Result<T, E>
```
Comment (stbrody) [inline event-svc/src/event/migration.rs:450]: man... Rust is cool

File: one/src/migrations.rs [hunk 2]
```diff
@@ -51,6 +51,11 @@ pub struct FromIpfsOpts {
     #[arg(long, env = "CERAMIC_ONE_LOCAL_NETWORK_ID")]
     local_network_id: Option<u32>,
 
+    /// Expect non sharded paths to blocks.
```
Comment (stbrody) [inline one/src/migrations.rs:54]: looks like some unrelated changes snuck into this PR?

**Review by stbrody — APPROVED** [review]

File: one/src/migrations.rs (hunk 2 above)
Comment (nathanielc) [reply one/src/migrations.rs]: Looks like git is just confused on the pre-merge, rebased to fix it.
